ios/NewExpensify/Chat.entitlements

```diff
@@ -1,14 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict>
```

Comment:
why these changes in ios folder?

Reply:
Pushed it by mistake (I needed it to run the app on device) - reverted